src/core/createOptIn.js

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
@Aaronius, I wonder if this module belongs inside the `/Privacy` component?

Reply:
Hmm. It would be the first time anything outside the components directory required a module located inside the components directory, aside from the component registration. I think I would prefer to keep it where it is since it more or less keeps component directories as black boxes, but if you feel like it should be in the privacy component directory, I can do it. I don't feel really strongly either way.